package.json

```diff
@@ -21,7 +21,7 @@
     "typescript": "~4.0.2"
   },
   "scripts": {
-    "test": "lab -a @hapi/code -t 100 -L -Y",
+    "test": "lab -a @hapi/code -t 100 -L -Y --types-test test/index.ts",
```

Comment:
This should not be needed. The `-Y' already does this.

Reply:
@kanongil Dropped